### DISCUSSION TITLE
install lib error

### DISCUSSION THREAD

**Discussion by fipoucat** [dtcenter/METplus#1526] [Installation] [answered]
I am having a c compiler can't create executable while comping Met on mac:

export MET_BUFRLIB=/Users/sarr/MET101/external_libs/lib
LDFLAGS = -Wl,--disable-new-dtags -Wl,-rpath,/Users/sarr/MET101/external_libs/lib:/Users/sarr/MET101/external_libs/bin/lib:/Users/sarr/MET101/external_libs/bin/lib:/Users/sarr/MET101/external_libs/lib:/Users/sarr/MET101/external_libs/lib:/usr/local/lib:/Users/sarr/MET101/external_libs/bin/lib -Wl,-rpath,/Users/sarr/MET101/external_libs/lib:/Users/sarr/MET101/external_libs/lib -L/Users/sarr/MET101/external_libs/lib -L/Users/sarr/MET101/external_libs/lib -L/Users/sarr/MET101/external_libs/bin/lib
cd /Users/sarr/MET101/met-10.1.0
./configure --prefix=/Users/sarr/MET101 --bindir=/Users/sarr/MET101/bin BUFRLIB_NAME=-lbufr GRIB2CLIB_NAME=-lgrib2c --enable-grib2 --enable-python  > configure.log 2>&1
configure returned with non-zero (77) status

**Comment by j-opatz** [comment]
Hello,

It looks like you initially posted a question, but your latest edit removed it. Do you still need help related to installation of MET? I can delete this discussion if you meant to completely remove your question as well.

**Comment by fipoucat** [comment, marked as answer]
I still have a problem in installing Met and Metplus and was editing the title and the content to move it to issues. I am trying to reedit it again with exact content

**Reply by j-opatz** [reply]
Thank you for providing the clarification.

For the time being, let's keep this in the Discussions section, as we might be able to help you with the issue without relying on code changes. We typically reserve Github issue creation for problems that will require changes to the METplus repository and try to resolve any user environment-related problems before proceeding to that step.

I'm going to tag @jprestop here, a team member who can help resolve installation-related issues. They're currently out of the office, but they'll reply as soon as they're able. In the meantime, thank you for your patience. 

I also wanted to offer the suggestion that one of our engineers provided that pertains to your previous problem of `sed` usage:

> They can replace any use of -r in the sed commands in the MET compile script with -E and that should fix their problem.

So if you do find any install commands using -r, a replacement -E may fix the issue.

**Comment by fipoucat** [comment]
The sed problem was solved by adding -E, it why I commented on the post which was deleted. It was after that I encountered the c compiler issue I posted.
